bin/carto

```diff
@@ -11,16 +11,17 @@ var existsSync = require('fs').existsSync || require('path').existsSync
 
 var yargs = require('yargs')
             .usage("Usage: $0 <source MML file>")
-            .options('h', {alias:'help', describe:'Display this help message'})
-            .options('v', {alias:'version', boolean:true, describe:'Display version information'})
+            .options('a', {alias:'api', describe:'Specify Mapnik API version'})
             .options('b', {alias:'benchmark', boolean:true, describe:'Outputs total compile time'})
+            .options('f', {alias:'file', describe:'Outputs to the given file instead of stdout.'})
+            .options('h', {alias:'help', describe:'Display this help message'})
             .options('l', {alias:'localize', boolean:true, default:false, describe:'Use millstone to localize resources when loading an MML'})
             .options('n', {alias:'nosymlink', boolean:true, describe:'Use absolute paths instead of symlinking files'})
-            .options('a', {alias:'api', describe:'Specify Mapnik API version'})
-            .options('f', {alias:'file', describe:'Outputs to the given file instead of stdout.'})
             .options('o', {alias:'output', describe:'Specify output format (mapnik, json)', default:'mapnik'})
+            .options('p', {alias:'profile', boolean:true, default:false, describe:'Generate v8 profile data that can be loaded in chromium/chrome\'s profiler for inspection.'})
             .options('q', {alias:'quiet', boolean:true, default:false, describe:'Do not output any warnings'})
-            .options('ppi', {describe:'Pixels per inch used to convert m, mm, cm, in, pt, pc to pixels', default:90.714});
+            .options('ppi', {describe:'Pixels per inch used to convert m, mm, cm, in, pt, pc to pixels', default:90.714})
```

Comment:
heeh, I missed that. in any case the reordering could be removed of the PR if you want.